mscore/preferences.cpp

```diff
@@ -501,14 +501,15 @@ void Preferences::read()
       dir.mkpath(myImagesPath);
       dir.mkpath(myTemplatesPath);
       dir.mkpath(myPluginsPath);
-      foreach (QString path, mySoundfontsPath.split(";"))
+      for (QString path: mySoundfontsPath.split(";"))
```

Comment:
you miss a space here before :
I'm pretty sure all these space modification will not apply to 2.1